server/controllers/chartsController/index.js

```diff
@@ -60,14 +61,29 @@ export const graphDataController = async (
       const shouldCountSubject = isVariableValueEmpty
         ? subjectDayData.every((day) => day[chart.variable] === value)
         : subjectDayData.some((dayData) => dayData[chart.variable] == value)
+      const getVariableValueCount =
+        isVariableValueEmpty && shouldCountSubject
+          ? 1
+          : calculateVariableCountsFromSubjectDayData(
+              subjectDayData,
+              chart.variable,
+              value
+            )
```

Comment:
Do you think this naming would better describe what this is? It seems to be the number of days the subject has the variable value?

```suggestion
      const subjectVariableDayCount =
        isVariableValueEmpty && shouldCountSubject
          ? 1
          : calculateSubjectVariableDayCount(
              subjectDayData,
              chart.variable,
              value
            )
```

Reply:
Thank you for this the name update is helpful.

https://github.com/AMP-SCZ/dpdash/pull/260/commits/47f214a9fd64edcfca08b4b6b94fa57f4b13ff12